packages/juggler/test/unit/legacy/legacy-juggler-bridge.ts

```diff
@@ -0,0 +1,165 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: @loopback/juggler
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+
+import {jugglerModule, bindModel, DataSource, juggler, DefaultCrudRepository}
+  from '../../../src/legacy-juggler-bridge';
+
+describe('legacy loopback-datasource-juggler', () => {
+  let ds: juggler.DataSource;
+
+  before(function() {
+    ds = new DataSource({
+      name: 'db',
+      connector: 'memory',
+    });
+    expect(ds.settings.name).to.eql('db');
+    expect(ds.settings.connector).to.eql('memory');
+  });
+
+  it('creates models', () => {
+    /* tslint:disable-next-line:variable-name */
+    const Note = <typeof juggler.PersistedModel>
```

Comment:
Can we provide type definitions for legacy API, so that this type cast is not necessary? Also my impression is that `<TYPE>` casting is considered as a bad practice these days and `as TYPE` should be used instead.